hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/TargetBeanQuery.java

```diff
@@ -180,12 +185,12 @@ public int size() {
         final long totSize = getTargetManagement().count();
         long size;
         if (null != targetFilterQueryId) {
-            size = getTargetManagement().countByTargetFilterQuery(targetFilterQueryId);
-        } else if (!isAnyFilterSelected()) {
+            TargetFilterQuery targetFilterQuery = getTargetFilterQueryManagement().getById(targetFilterQueryId);
```

Comment:
final

Reply:
`final` not possible here (even if I would like it 😏):
```java
final long totSize = getTargetManagement().count();
        long size;
        if (null != targetFilterQueryId) {
            TargetFilterQuery targetFilterQuery = getTargetFilterQueryManagement().getById(targetFilterQueryId);
            size = getTargetQueryExecutionManagement().countByQuery(targetFilterQuery.getQuery());
        } else if (filter.isEmpty()) {
            size = totSize;
        } else {
            size = getTargetManagement().countByFilters(getFilter());
        }
```